test/EndToEnd/tests/InstallPackageTest.ps1

```diff
@@ -2984,4 +2984,4 @@ function Test-InstallPackageWithRootNamespaceInPPFile {
     $path = (Get-ProjectItemPath $p foo.cs)
     $content = [System.IO.File]::ReadAllText($path)
     Assert-True ($content.Contains("namespace testProject"))
-}
+}
```

Comment:
😞 your text editor doesn't respect `.editorconfig`, which we've configured to save files with a new line character on the last line of the file.  If you're using VSCode, you need an extension to load `.editorconfig` settings.

In any case, there are no other changes to this file, so please revert it.

Reply:
Fixed. Thanks~